src/windows/appw.c

```diff
@@ -993,9 +1001,12 @@ static LRESULT app_custom_hwnd_proc(struct window *ctx, HWND hwnd, UINT msg, WPA
 			pkt.pkY = position.y;
 
 			wintab_on_packet(app->wintab, &evt, &pkt, focused_window->window);
-			
+
+			if (app->pen_enabled && evt.pen.flags & MTY_PEN_FLAG_TOUCHING)
+				app->pen_type = MTY_PEN_TYPE_WACOM;
```

Comment:
So, only when wintab is enabled, and the pen gets touched, it will be a wacom tablet, there is no other way this could be something else?